packages/cursorless-vscode-e2e/src/suite/fixtures/scopes/javascript/name.field.scope

```diff
@@ -0,0 +1,89 @@
+class MyClass {
+  value = 0;
+}
+---
+
+[#1 Content] = 0:6-0:13
+0| class MyClass {
+        >-------<
+1|   value = 0;
+
+2| }
```

Comment:
This one is another good argument for fine-grained scopes

Reply:
Definitely